### ISSUE TITLE
Better place name label hierarchy & placement

### ISSUE THREAD

**Issue by clhenrick** [stamen/terrain-classic#81]
Believe labels for certain places are being sized based on population data, this should probably not be the sole factor for label prominence. Perhaps it would be helpful to pull `scalerank` from Natural Earth's populated places to be included with Aries and use that instead?

For example the Seattle area at z11:
![](https://cloud.githubusercontent.com/assets/6434485/16747677/fe9f9d16-4774-11e6-9230-7b7bfd69de0e.png)

And the SF Bay Area at z11, the label for SF should probably be more prominent than Oakland, though it has a smaller population:
![screen shot 2016-07-11 at 10 12 15 pm](https://cloud.githubusercontent.com/assets/161748/16755750/b19d64ae-47b4-11e6-94fb-8d5d2127e8ff.png)

**Comment by geoglrb** [comment]
On the top map, Eastgate has population <5000 and is labeled. Seattle has population >500000 and is not labeled. The problem may not be correlated with population, so it might be more of a bug...

**Comment by clhenrick** [comment]
Yes it's likely to do with Seattle's label falling on or very close to a tile edge which may be why it's not being rendered.
